src/schema/posts.ts

```diff
@@ -276,6 +279,10 @@ export const typeDefs = /* GraphQL */ `
     """
     sourceId: String
     """
+    Source
+    """
+    source: Source
+    """
```

Comment:
Then here it should be
```suggestion
    Source
    """
    source: Source!
    """
```